src/components/templates/NavBar.tsx

```diff
@@ -43,4 +49,4 @@ function NavBar (){
   )
 }
 
-export default NavBar
\ No newline at end of file
+export default NavBar
```

Comment:
Is this the PR to fix the `export default`s too?

Reply:
It wasn't, no.